src/test/java/net/minecraftforge/debug/DataGeneratorTest.java

```diff
@@ -32,7 +32,9 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import net.minecraft.advancements.criterion.InventoryChangeTrigger;
 import net.minecraft.block.*;
+import net.minecraft.data.*;
```

Comment:
Did you add this?
Try to avoid wildcard imports.

Reply:
Intellij does this automatically for me i will change this for forge